packages/models/src/manufacturer/manufacturer.ts

```diff
@@ -1,6 +1,7 @@
 import { ManufacturerSeo } from "./manufacturerSeo";
 
 export interface Manufacturer {
+    id?: string;
```

Comment:
remove the field, because this is not relevant to models, this is only being used for seeder. so you can import this interface in seeder and then extend it to add the desired properties

Reply:
Yes, updated using extends in seeder